pkg/crawler/main.go

```diff
@@ -120,8 +123,10 @@ func walkSystems(rf_systems []*redfish.ComputerSystem, rf_chassis *redfish.Chass
 	for _, rf_computersystem := range rf_systems {
 		system := InventoryDetail{
 			URI:            baseURI + "/redfish/v1/Systems/" + rf_computersystem.ID,
+			UUID:           rf_computersystem.UUID,
 			Name:           rf_computersystem.Name,
 			Manufacturer:   rf_computersystem.Manufacturer,
+			SystemType:     string(rf_computersystem.SystemType),
```

Comment:
Any reason you are calling `string()` here? I notice the other right-hand values above and below don't have itand they are of type `string`.

Reply:
Not sure, since I didn't add that myself. I think it was added automatically by `gofmt`.

Reply:
After looking at the `gofish` code, I think it's to cast from `SystemType` to a `string`, since `SystemType` is the actual type of `rf_computersystem.SystemType`.